tests/common/db/banners.py

```diff
@@ -0,0 +1,31 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import date, timedelta
+
+from factory import fuzzy
+
+from warehouse.banners.models import Banner
+
+from .base import FuzzyUrl, WarehouseFactory
+
+
+class BannerFactory(WarehouseFactory):
+    class Meta:
+        model = Banner
+
+    name = fuzzy.FuzzyText(length=12)
+    text = fuzzy.FuzzyText(length=30)
+    link_url = FuzzyUrl()
```

Comment:
I used `factory.fuzzy` to respect the same pattern as in the other `Factory` objects. Although I'm in favor of such refactoring, I think it would be better if we apply it to all the factories so we don't end up with 2 ways of generating test data. What do you think about this? Should we open an issue to address this refactoring?

I think it would be a nice issue for first time contributors since it would be a refactoring in the tests and, so, with an easy to verify if the code works.

Reply:
I agree. This is a great idea for filing as an Issue and flagging "good first"

Reply:
Such refactoring is now detailed in https://github.com/pypa/warehouse/issues/9707